spec/motion-spec/should_spec.rb

```diff
@@ -1,34 +1,31 @@
 # -*- encoding : utf-8 -*-
-motion_require '../spec_helper'
 
 describe "#should shortcut for #it('should')" do
-
-  should "be called" do
+  should 'be called' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.